client/lib/plans/test/plan-lookups.js

```diff
@@ -233,10 +245,31 @@ describe( 'isWpComBusinessPlan', () => {
 		expect( isWpComBusinessPlan( PLAN_JETPACK_PERSONAL_MONTHLY ) ).to.equal( false );
 		expect( isWpComBusinessPlan( PLAN_PREMIUM ) ).to.equal( false );
 		expect( isWpComBusinessPlan( PLAN_JETPACK_PREMIUM ) ).to.equal( false );
+		expect( isWpComBusinessPlan( PLAN_ECOMMERCE ) ).to.equal( false );
 		expect( isWpComBusinessPlan( 'non-exisWpComting plan' ) ).to.equal( false );
 	} );
 } );
 
+describe( 'isWpComEcommercePlan', () => {
+	test( 'should return true for eCommerc plans', () => {
+		expect( isWpComEcommercePlan( PLAN_ECOMMERCE ) ).to.equal( true );
+		expect( isWpComEcommercePlan( PLAN_ECOMMERCE_2_YEARS ) ).to.equal( true );
+	} );
+	test( 'should return false for non-business plans', () => {
+		expect( isWpComEcommercePlan( PLAN_JETPACK_BUSINESS ) ).to.equal( false );
+		expect( isWpComEcommercePlan( PLAN_JETPACK_BUSINESS_MONTHLY ) ).to.equal( false );
+		expect( isWpComEcommercePlan( PLAN_PERSONAL ) ).to.equal( false );
+		expect( isWpComEcommercePlan( PLAN_PERSONAL_2_YEARS ) ).to.equal( false );
+		expect( isWpComEcommercePlan( PLAN_JETPACK_PERSONAL ) ).to.equal( false );
+		expect( isWpComEcommercePlan( PLAN_JETPACK_PERSONAL_MONTHLY ) ).to.equal( false );
+		expect( isWpComEcommercePlan( PLAN_PREMIUM ) ).to.equal( false );
+		expect( isWpComEcommercePlan( PLAN_JETPACK_PREMIUM ) ).to.equal( false );
+		expect( isWpComEcommercePlan( PLAN_BUSINESS ) ).to.equal( false );
+		expect( isWpComEcommercePlan( PLAN_BUSINESS_2_YEARS ) ).to.equal( false );
+		expect( isWpComEcommercePlan( 'non-exisWpComting plan' ) ).to.equal( false );
```

Comment:
There seems to be a typo here

Reply:
oh, I just meant `non-exisWpComting plan` vs `non-existing WpCom plan`

Reply:
Yeah, that was there before, I used the same thing just to have it consistent:
https://github.com/Automattic/wp-calypso/blob/master/client/lib/plans/test/plan-lookups.js#L184

Reply:
TBH It's a typo :-) and it definitely works both ways